src/aliceVision/sfm/utils/alignment.hpp

```diff
@@ -178,6 +178,21 @@ inline void applyTransform(sfmData::SfMData& sfmData,
   }
 }
 
+/**
+ * @brief Compute the new coordinate system in the given SfM so that the mean
+ * of the camera centers is the origin of the world coordinate system,
+ * a dominant Y axis is defined based on the X axis of all cameras,
+ * and the scale is set so that the optical centers RMS is "1.0".
+ * (Hartley-like normalization, p.180)
```

Comment:
Can you please provide a usable reference (p.180 of what?)

Reply:
Richard Hartley and Andrew Zisserman. 2003. Multiple View Geometry in Computer Vision (2nd. ed.). Cambridge University Press, USA.
It's the bible of computer vision, so you only cite the page... ;-)